.github/CODEOWNERS

```diff
@@ -77,3 +77,40 @@ Makefile                    @getsentry/owners-sentry-dev
 /tests/sentry/snuba/test_tasks.py                               @getsentry/workflow
 /tests/snuba/snuba/test_query_subscription_consumer.py          @getsentry/workflow
 ### Endof Issue/Metric Alerts ###
+
+
+### Perfomance/Discover ###
+/src/sentry/snuba/                                              @getsentry/visibility
+/tests/sentry/snuba/test_discover.py                            @getsentry/visibility
+
+/src/sentry/discover/                                           @getsentry/visibility
+
+/src/sentry/utils/snuba.py                                      @getsentry/visibility
+
+/src/sentry/api/event_search.py                                 @getsentry/visibility
+/src/sentry/api/issue_search.py                                 @getsentry/visibility
```

Comment:
I'm ok with taking on issue search, but it is a bigger scope item.

Reply:
Oh I didn't realize the precedence rules. I assumed multiple people could own a file. I'll remove issue_search.

Reply:
Actually, maybe we should keep it. I don't know if there's a team that actually owns it right now.

Reply:
I don't think there is an alternate owner for issue search right now. We might as well take it as interim coverage.